cypress.config.ts

```diff
@@ -1,34 +1,54 @@
+import path from "path"
+
+import wp from "@cypress/webpack-preprocessor"
 import { defineConfig } from "cypress"
 
 export default defineConfig({
   chromeWebSecurity: false,
   viewportHeight: 768,
   viewportWidth: 1366,
   projectId: "nxbty1",
-
-  // NOTE: Cypress keeps 50 tests in memory by default;
-  // Unfortunately, this leads to OOM issues occasionally.
-  // This has been reduced to 5 to help with the OOM issues.
-  // Refer here: https://docs.cypress.io/guides/references/configuration#Global for details.
-  numTestsKeptInMemory: 5,
```

Comment:
Hey just checking, is this intended?

Reply:
@kishore03109 When running the tests, I didn't find any issues on my browser. But I will add this back just to be safe